Include/connect.php

```diff
@@ -1,5 +1,5 @@
 <?php
 DEFINE('DB_HOSTNAME', 'localhost');
-DEFINE('DB_DATABASE', 'personal_site');
+DEFINE('DB_DATABASE', 'Project');
 DEFINE('DB_USERNAME', 'root');
-DEFINE('DB_PASSWORD', 'root'); 
\ No newline at end of file
+DEFINE('DB_PASSWORD', 'root');
```

Comment:
You don't have to do this now but if you do want to deploy this site eventually, it's good practice to exclude this kind of file from getting saved in GitHub by 'git ignoring' it. It's fine for now I'm just mentioning this in case you want to look up git ignore or if you have any other questions